service/service.go

```diff
@@ -232,6 +248,16 @@ func (s *Service) loop(ctx context.Context, roundToResume *round) error {
 			s.executingRounds[round.ID] = struct{}{}
 
 			end := s.roundEndTime(round)
+
+			select {
+			case s.rounds <- RoundData{
+				ID:      round.ID,
+				Members: round.Members(),
+			}:
+			default:
+				logger.Debug("could not push round data, the receiving end might've hang")
+			}
```

Comment:
imo this should just be `servicedatabase.AddRound(round.ID, round.Members())`.

Should persisting a round fail if the DB is at the same moment persisting a proof or another round?